ChustaSoft.Tools.Authorization.TestBasic.WebAPI/Startup.cs

```diff
@@ -44,7 +45,9 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.RegisterAuthorization(_configuration, "d5ab5e3f5799445fb9f68d1fcdda3b9f")
-                .WithSqlServerProvider(BuildConnectionString());
+                .WithSqlServerProvider(BuildConnectionString());            
+
+            services.RegisterExternalAuthentication(_configuration);
```

Comment:
This should be performed automatically through the settings or through the builder.

Please check it

Reply:
Logic for external authentication registering has been extracted, and method has been overloaded to allow accept IConfiguration settings or authorization settings builder

Reply:
Thanks for have in mind! This was required.

However, one point is still missing, this line should not be required, instead of using this additional line in the configuration, shoud be performed automatically (Private method instead of public one) doing it just in case of having external configurations added to the Settings DTO